pipeline-modules/app-service/aws/docker-build-n-push.yaml

```diff
@@ -5,6 +5,7 @@ revision: 1
 displayName: Docker Build
 description: This plugin is for Docker build & push
 target: ""
+category: containerise
```

Comment:
Are these the categories that are going to show up in the UI? How are they going to map to what UX wants on the screen? i.e. what is "containerise"?

Reply:
according to the conversation with @pranaySinghDev, it looks like yes, this is supposed to map with categories in UX.